crates/release_plz/src/args/set_version.rs

```diff
@@ -29,6 +29,9 @@ pub struct SetVersion {
         value_parser = PathBufValueParser::new()
     )]
     config: Option<PathBuf>,
+    /// In a workspace update all member's who reference the crate's version.
+    #[arg(long)]
+    disable_dependant_updates: bool,
```

Comment:
are you sure "dependant" is the right term? 
![image](https://github.com/user-attachments/assets/3903ed47-00e2-44e6-a856-e7ace89536d4)